c7n/resources/discovery.py

```diff
@@ -0,0 +1,32 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.manager import resources
+from c7n import query
+
+
+@resources.register("aws.appdiscovery-agent")
+class AppdiscoveryAgent(query.QueryResourceManager):
+    class resource_type(query.TypeInfo):
+        service = "discovery"
+        enum_spec = ('describe_agents', 'agentsInfo', None)
+        arn = False
+        id = "agentId"
+        name = "hostName"
+
+
+
+@resources.register("aws.appdiscovery")
+class Appdiscovery(query.QueryResourceManager):
```

Comment:
this is one of the stranger api schemas I've seen, seems to bypass schema modeling if favor of string: string maps with implicitly used keys.

Reply:
due to the oddity of the api on this, I'd feel its probably better to remove this resource for now till we know the concrete values on it.